src/test/java/spoon/getExecutableDeclarationBugTest/testClasses/ChangePasswordValidator.java

```diff
@@ -0,0 +1,43 @@
+package pt.ist.socialsoftware.edition.ldod.validator;
```

Comment:
This package name is incorrect, it should be declared as `package spoon.getExecutableDeclarationBugTest.testClasses;`